src/unitxt/api.py

```diff
@@ -198,8 +198,12 @@ def load_dataset(
     ).with_transform(loads_instance)
 
 
-def evaluate(predictions, data) -> EvaluationResults:
-    return _compute(predictions=predictions, references=data)
+def evaluate(
+    predictions, dataset: Union[Dataset, IterableDataset], data=None
+) -> EvaluationResults:
+    if data is not None:
+        dataset = data  # for backward compatibility
```

Comment:
What if both are not None?

Reply:
I wanted to rename `data` to `dataset` to have consistent name with load_dataset and create_dataset and DatasetRecipe but yet have backward compatibility for code using `data=`